functions/Get-DbaLastBackup.ps1

```diff
@@ -0,0 +1,158 @@
+﻿Function Get-DbaLastBackup
+{
+<#
+.SYNOPSIS
+Get date/time for last known backups
+
+.DESCRIPTION
+Retrieves and compares the date/time for the last known backups, as well as the creation date/time for the database.
+
+
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER Databases
+Return information for only specific databases
+
+.PARAMETER Exclude
+Return information for all but these specific databases
+
+.PARAMETER Detailed
+Shows detailed information
+
+.NOTES
```

Comment:
Oops, I pasted wrong. Sorry, that was not my intention. I have done this is Get-DbaPermission too. Sorry again.